gcp_variant_transforms/libs/bigquery_vcf_schema_test.py

```diff
@@ -195,7 +195,7 @@ def test_variant_merger_modify_schema(self):
         ('I1', Info('I1', 1, 'String', 'desc', 'src', 'v')),
         ('IA', Info('IA', field_counts['A'], 'Integer', 'desc', 'src', 'v'))])
     formats = OrderedDict([('F1', Format('F1', 1, 'String', 'desc'))])
-    header_fields = vcf_header_parser.HeaderFields(infos, formats)
+    header_fields = vcf_header_io.VcfHeader(infos=infos, formats=formats)
```

Comment:
Good question. For the schema (and our code in general), we just care about INFO and FORMAT fields and ignore the rest of the fields. The rest are included as part of the 'representative header' for completeness (in case we end up using them at some point in the future and/or if they're used by the parser).